CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## Unreleased
 
-## 1.32.0 - (September 1, 2020)
+## 1.33.0 - (September 3, 2020)
 
 * Added
   * Added `aria` role to announce error boundary error text.
+
+## 1.32.0 - (September 1, 2020)
+
+* Added
```

Comment:
The changelog looks a little different than usual because the change entry got auto merged into the previous release accidentally. I had to manually move it up before running the change log updater.